flocker/node/_loop.py

```diff
@@ -501,15 +523,23 @@ def got_local_state(local_state):
                 self.configuration, self.cluster_state, local_state
             )
             if isinstance(action, NoOp):
+                # If we have converged, we need to reset the sleep delay
+                # in case there were any incremental back offs while
+                # waiting to converge.
+                self._unconverged_sleep.reset_delay()
                 # We've converged, we can sleep for NoOp's sleep duration.
                 # We add some jitter so not all agents wake up at exactly
                 # the same time, to reduce load on system:
+                self._unconverged_sleep.reset_delay()
```

Comment:
nit: I don't think there is benefit in calling this twice, I suspsect some sort of merge error.